rpcs3/Crypto/unpkg.h

```diff
@@ -313,7 +313,8 @@ class package_reader
 
 	struct install_entry
 	{
-		typename std::map<std::string, install_entry*>::value_type* weak_reference{};
```

Comment:
It was an optimization which was perfect here.

Reply:
didn't work with my code. But I can try with the new code